### PR TITLE
Use FROM docs/base:latest again

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

### CODE REVIEW

**Comment by dnephin** [comment]
LGTM

**Comment by SvenDowideit** [comment]
@moxiegirl

File: docs/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,8 @@ RUN svn checkout https://github.com/kitematic/kitematic/trunk/docs /docs/content
 RUN svn checkout https://github.com/docker/tutorials/trunk/docs /docs/content/tutorials
 RUN svn checkout https://github.com/docker/opensource/trunk/docs /docs/content
 
+ENV PROJECT=compose
 # To get the git info for this repo
 COPY . /src
 
-COPY . /docs/content/compose/
+COPY . /docs/content/$PROJECT/
```
Comment (moxiegirl) [inline docs/Dockerfile]: What is the value of adding the `ENV` to this?

**Comment by moxiegirl** [comment]
LGTM

**Pull request merged**